vpp/vaapipostprocess_scaler.h

```diff
@@ -48,6 +48,26 @@ class VaapiPostProcessScaler : public VaapiPostProcessBase {
         BufObjectPtr filter; //send to va;
     };
 
+    struct ColorBalanceModeRange {
+        VppColorBalanceMode mode;
+        VAProcFilterValueRange range;
+    };
+    struct ColorBalanceCompare {
+        bool operator()(const ColorBalanceModeRange& mr1, const ColorBalanceModeRange& mr2) const
+        {
+            return (mr1.mode < mr2.mode) ? true : false;
+        }
+    };
+
+    typedef std::set<ColorBalanceModeRange, ColorBalanceCompare> ColorBalanceList;
+    typedef std::set<ColorBalanceModeRange, ColorBalanceCompare>::iterator ColorBalanceItr;
+    struct ColorBalanceParams {
+        VppColorBalanceMode mode; //set by user
```

Comment:
I just put VppColorBalanceMode and VAProcFilterValueRange together, so I can find out the range value via VppColorBalanceMode. 
But I will save them in a Map, according to your suggestion.
Thanks @xuguangxin .